tv_apps/smart-home/js/home.js

```diff
@@ -516,6 +515,20 @@
       var focus = this.spatialNavigator.getFocusedElement();
 
       if (!(focus.CLASS_NAME == 'XScrollable' && focus.move(key))) {
```

Comment:
This is unrelated to your patch... but I think we can use
```
if (focus.CLASS_NAME == 'XScrollable' && focus.move(key)) {
  return;
}
```
then put your rest conditions to reduce indents.